src/components/nested-table-view/common/editable-table-cell.tsx

```diff
@@ -45,20 +47,49 @@ export const EditableTableCell = observer(function EditableTableCell(props: IPro
     }
   };
 
+  const handleCellClick = (e: React.MouseEvent<HTMLDivElement>) => {
+    e.preventDefault();
+    if (isEditing) return;
+    const isExtending = e.altKey || e.metaKey;
```

Comment:
What about `shiftKey`?

Reply:
I didn't implement shift select because it required more logic (finding the index of the previously selected case and the currently selected case, etc.)